python/cudf/udf_cpp/shim.cu

```diff
@@ -662,3 +664,15 @@ make_definition_idx(BlockIdxMax, int64, int64_t);
 make_definition_idx(BlockIdxMax, float64, double);
 #undef make_definition_idx
 }
+
+__device__ void udf_str_dtor(void* udf_str, size_t size, void* dtor_info)
+{
+  udf_string* ptr = reinterpret_cast<udf_string*>(udf_str);
+  ptr->clear();
```

Comment:
Given @davidwendt's comment above, then I think this should be:
```suggestion
  ptr->~udf_string();
```

And I think then there' no explanatory comment needed, because it's clear this is just a Numba shim for the destructor.